Sources/Control/Extensions/MPVolumeView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  MPVolumeView.swift
+//  ControlKit
+//
+
+import MediaPlayer
+import OSLog
+
+extension MPVolumeView {
+    
+    static var volume: Float {
+        get {
+            shared.slider?.value ?? 0
+        }
+        set {
+            shared.slider?.value = newValue
+        }
+    }
+    
+    static func increaseVolume(_ amount: Float) {
+        guard volume <= 1 else {
+            log.warning("Volume is already at max")
+            return
+        }
+        volume = max(1, volume + amount)
```

Comment:
It appears that the parameters used in the functions `increaseVolume` and `decreaseVolume` have been reversed.

`increaseVolume` should utilize the `min`, while `decreaseVolume` should utilize the `max`.

Reply:
What a catch! ⚾️